boards/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -113,7 +113,7 @@ class BmsStateMachineTest : public testing::Test
     struct RgbLedSequence *   rgb_led_sequence;
 };
 
-TEST_F(BmsStateMachineTest, check_init_state_is_broadcasted_over_can)
+TEST_F(BmsStateMachineTest, check_init_state_is_broadcasted_over_can_BMS_31)
```

Comment:
What do you think about `BMS_31__check_init_state_is_broadcasted_over_can` or `check_init_state_is_broadcasted_over_can__BMS_31`? Just trying to make it a bit more clear that `BMS_31` is somewhat seperate.

Reply:
I think that's a good point, let me add the double underscore.